pkg/interfaces/contracts/vault/IUnbalancedLiquidityInvariantRatioBounds.sol

```diff
@@ -4,6 +4,12 @@ pragma solidity ^0.8.24;
 
 /**
  * @notice Return the minimum/maximum invariant ratios allowed during an unbalanced liquidity operation.
+ * @dev The Vault does not enforce bounds on invariant ratios; `IBasePool` implements this interface to ensure
+ * that new pool developers think about and set these bounds according to their specific pool type.
```

Comment:
hmm maybe I'd rephrase this a bit.

The vault does enforce the bounds, based on what the pool sets as bounds. It just doesn't enforce any 'extra' or baseline bounds.